### PR TITLE
CUMULUS-3737: Merge v18.3.1 updates into master

### PR DESCRIPTION
**Pull request by paulpilone**
**Summary:** Summary of changes

Addresses [CUMULUS-3737: Release 18.3.1](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3737)

## Changes

* Updates CL to include changes from the `release-18.3.x` branch after releasing v18.3.1
* Updates CONTRIBUTORS to reflect 18.3.1
* Includes new versioned docs for 18.3.1

## PR Checklist

- [x] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

**Comment by paulpilone** [comment]
I had to manually edit the CL in order to get the tickets to correctly reflect v18.3.1 and the current state of master. I think this was because there were multiple commits on the release-18.3.x branch that updated the CL and one included many other changes. It should be right in this PR but asking for a thorough review of it.

File: CHANGELOG.md [hunk 1]
```diff
@@ -84,9 +120,14 @@ of the migration as it runs, you can view the CloudWatch logs for your async
 operations (e.g. `PREFIX-AsyncOperationEcsLogs`).
 
 #### CUMULUS-3779 async_operations Docker image version upgrade
-  
-The `async_operations` Docker image has been updated to support Node v20 and `aws-sdk` v3. Users will need to bump
-the version tag of `async_operations` to at least 52 if using the Docker image.
+
+The `async-operation` Docker image has been updated to support Node v20 and `aws-sdk` v3. Users of the image will need
```
Comment (Jkovarik) [inline CHANGELOG.md:124]: 👍🏻

**Review by Jkovarik — APPROVED** [review]
Looks good 👍🏻

**Comment by paulpilone** [comment]
@Jkovarik I had to resolve a small merge conflict from a commit pushed to master a few minutes ago. It was a small one and just involved moving the CUMULUS-3787 entry was properly placed in Unreleased -> Fixed.

**Review by Jkovarik — APPROVED** [review]

**Pull request merged**